src/pybind/mgr/dashboard_v2/CMakeLists.txt

```diff
@@ -9,10 +9,16 @@ add_custom_target(mgr-dashboard_v2-test-venv
 add_dependencies(tests mgr-dashboard_v2-test-venv)
 
 if(WITH_MGR_DASHBOARD_V2_FRONTEND AND NOT CMAKE_SYSTEM_PROCESSOR MATCHES "aarch64|AARCH64|arm|ARM")
```

Comment:
Can we also change the condition to match `x86_64|amd64` explicitly instead of checking if it's not arm?

Reply:
i think #20752 is trying to address this.

Reply:
ah, right!